### DISCUSSION TITLE
Firmware update after Patching.

### DISCUSSION THREAD

**Discussion by BooBoo124** [Mr-MIBonk/M.I.B._More-Incredible-Bash#487] [Q&A]
I have patched MHI2Q-AUG22-P4212 successfully with MIB a while back. If I up date to the latest firmware MHI2Q_ER_AUG22_P5092 will it remove 23000ee fec, and I need to patch again or can I update and patch in one go, or do I need to update and then patch again after? Obvs disabling the B&O update via user defined. 

Cheers

**Comment by visarkumnova** [comment]
No it wont remove 23000ee

**Comment by BooBoo124** [comment]
So I won't need to patch again then. But obviously remember to set user defined to protect my B&O amp

**Comment by BooBoo124** [comment]
I did update 23000ee was invalid and green menu and MIB had gone
Had to repatch

**Comment by visarkumnova** [comment]
You need to insert the SD Card on slot 1 to have MIB without SD Card you don’t have MIB . And for invalid license 023000ee what error did the appear can you send me a photo to see.


Sent from my iPhone

On Jun 22, 2024, at 2:40 PM, BooBoo124 ***@***.***> wrote:

﻿

I did update 23000ee was invalid and green menu and MIB had gone
Had to repatch

**Reply by BooBoo124** [reply]
I couldn't get green menu so no mib 23000ee invalid
That's all it said. MiB card was in SD1

**Comment by visarkumnova** [comment]
Did you uninstall the MIB , if you uninstalled you need to activate with script green menu and install again MIB . Here the link for script just on zip on SD Card and insert on Slot 1 wait for 2-3 min and than install the MIB again

<https://mega.nz/#!J4IwSSrC!gaEDqasks3vTeY2VDTu6moPod6nUi_l6gDb77BKMZZU>
[b41537c0eae056cfe5ab05902fca322b.png]
MEGA<https://mega.nz/#!J4IwSSrC!gaEDqasks3vTeY2VDTu6moPod6nUi_l6gDb77BKMZZU>
mega.nz<https://mega.nz/#!J4IwSSrC!gaEDqasks3vTeY2VDTu6moPod6nUi_l6gDb77BKMZZU>



Sent from my iPhone

On Jun 22, 2024, at 2:49 PM, BooBoo124 ***@***.***> wrote:

﻿

I could get green menu so no mib 23000ee invalid
That's all it said. MiB card was in SD1
